crates/bevy_render/src/view/mod.rs

```diff
@@ -129,40 +169,209 @@ impl ExtractedView {
     }
 }
 
-/// Configures basic color grading parameters to adjust the image appearance. Grading is applied just before/after tonemapping for a given [`Camera`](crate::camera::Camera) entity.
-#[derive(Component, Reflect, Debug, Copy, Clone, ShaderType)]
+/// Configures filmic color grading parameters to adjust the image appearance.
+///
+/// Grading is applied just before/after tonemapping for a given
```

Comment:
As a potential user of color grading I'm not sure how to interpret

> just before/after

could this explanation be expanded a bit?

Reply:
Done.